src/quo/components/wallet/progress_bar/component_spec.cljs

```diff
@@ -63,7 +63,7 @@
                     :width           8
                     :borderRadius    3
                     :borderColor     colors/neutral-80-opa-5
-                    :backgroundColor (colors/custom-color (:customization-color props) 50)})))
+                    :backgroundColor (colors/resolve-color (:customization-color props) 50)})))
```

Comment:
@mmilad75 there is many misuses of resolve color here. the second param is `theme` not 50/60 values 👌 . 
It should be "(colors/resolve-color `color` `theme` `opacity(optional)`)"

Reply:
Done